engine/api/services/dao.go

```diff
@@ -62,7 +62,11 @@ func get(ctx context.Context, db gorp.SqlExecutor, q gorpmapping.Query) (*sdk.Se
 // LoadAll returns all services in database.
 func LoadAll(ctx context.Context, db gorp.SqlExecutor) ([]sdk.Service, error) {
 	query := gorpmapping.NewQuery(`SELECT * FROM service`)
-	return getAll(ctx, db, query)
+	services, err := getAll(ctx, db, query)
+	if err != nil {
+		return nil, err
+	}
+	return withServiceStatus(ctx, db, services)
```

Comment:
it could be nice to pass a loader optional function as we are doing in other packages